src/service/resources-impl.js

```diff
@@ -205,7 +205,7 @@ export class ResourcesImpl {
      */
     this.intersectionObserverCallbackFired_ = false;
 
-    if (isExperimentOn(this.win, 'bento')) {
+    if (isExperimentOn(this.win, 'intersect-resources')) {
```

Comment:
Since the experiment is already at 0.
```suggestion
   // TODO(#33262): cleanup
    if (false) {
```